native_client/python/client.py

```diff
@@ -35,6 +38,13 @@
 # Size of the context window used for producing timesteps in the input vector
 N_CONTEXT = 9
 
+def convert_samplerate(audio_path):
+    temp_dir = tempfile.mkdtemp()
+    output_path = os.path.join(temp_dir, 'output.wav')
```

Comment:
Checking the manpage of `sox` shows that it can deal with `-` as a special filename for `stdin` / `stdout`. It would be perfect if you can change that :)